### ISSUE TITLE
How to generate stage1_metadata.csv with keras? 

### ISSUE THREAD

**Issue by bruceyang2012** [neptune-ml/data-science-bowl-2018#56]
Hi, when I run mask-rcnn notebook, I'm stuck on that there is no "stage1_metadata.csv". I tried to run your pytorch code to generate it, but  I encountered a lot of problems because I am not familiar with pytorch. And it seems that windows can not support pytorch very well. 
Therefore, could you tell me how to generate "stage1_metadata.csv" with keras? I wonder if you could help me with this.

**Comment by jakubczakon** [comment]
If you are running it locally then yes you need to generate metadata via:

```bash
neptune run -- prepare_metadata
```
or
```bash
python run main.py -- prepare_metadata
```
just make sure to change the paths in the `neptune.yaml` accordingly

```yaml
...
# Local Environment
  data_dir:                        /path/to/data
  meta_dir:                        /path/to/data
  masks_overlayed_dir:             /path/to/masks_overlayed
  contours_overlayed_dir:          /path/to/contours_overlayed
  contours_touching_overlayed_dir: /path/to/contours_touching_overlayed/
  centers_overlayed_dir:           /path/to/centers_overlayed
  experiment_dir:                  /path/to/work/dir
```

**Comment by kamil-kaczmarek** [comment]
@bruceyang2012 did you manage to generate `stage1_metadata.csv`. If so, I will close this issue.

**Comment by bruceyang2012** [comment]
Thanks for your kind, I will try it this weekend.

**Comment by kamil-kaczmarek** [comment]
@bruceyang2012 Did you manage to generate `stage1_metadata.csv`?

**Comment by kamil-kaczmarek** [comment]
closing this. @bruceyang2012 please re-open in case of any further questions.
